app/components/agenda/modal.hbs

```diff
@@ -1,14 +1,32 @@
 {{#wu-modal
```

Comment:
If possible we should use an appuniversum equivalent of wu-modal. if it doens't exist, we should consider creating one.

Reply:
It does exist, but it doesn't allow parent components to open/close it yet. Instead it renders it's own button, so it's pretty awkward to use atm, there is a TODO outstanding for this
https://appuniversum.github.io/ember-appuniversum/docs/components/au-modal
Will update it with octane syntax